README.md

```diff
@@ -365,66 +398,83 @@ ExternalProject_Add(Nabla
     TEST_COMMAND      ""
 )
 
-# replace with whatever you need ${PROJECT_NAME}
-target_include_directories(${PROJECT_NAME}
+function(addNablaModule target)	
```

Comment:
why doesn't our cmake (or one of the smake include files) provide this function?

Reply:
because I haven't been asked ( ͡° ͜ʖ ͡°)

Reply:
we can add this one

Reply:
maybe in a separate file in the `/cmake/` so that its available (without adding subdirectory the whole of Nabla) even if someone uses ExternalProject

Reply:
I prefer `/cmake/build/`, it would be a mess to put it here 
![image](https://user-images.githubusercontent.com/34793522/125763579-a979ba76-8f68-437a-866d-b3b57d02c382.png)

Reply:
okay, btw I removed `import`

Reply:
ah okay, and done